powershell/utils/http-operation.ts

```diff
@@ -298,3 +299,20 @@ export enum ParameterLocation {
 //     this.apply(initializer);
 //   }
 // }
+
+export function hasValidBodyParameters(
```

Comment:
Seem this function is defined twice.

Reply:
The parameter are different `Operation` class